tests/chicken/test-support.scm

```diff
@@ -0,0 +1,4 @@
+(define-syntax test-equal
```

Comment:
Both the Chicken test egg and Chibi test libraries use `test`, I prefer to keep that.

Reply:
CHICKEN core uses `test-equal` and every time we update Irregex the tests have to be "ported", so I'd be fine with this change.

Reply:
Why not just use this definition, swapping `test` and `test-equal`?